test/plugin/agent-test-tools/pom.xml

```diff
@@ -35,9 +35,9 @@
     <packaging>pom</packaging>
 
     <properties>
-        <agent-test-tools.version>7f20775e0631356c4823d9372b09d653db0e6540</agent-test-tools.version>
+        <agent-test-tools.version>8a09b6f16b0b3fb535ca515437ef107042aaae45</agent-test-tools.version>
         <agent-test-tools.workingDirectory>${project.basedir}/target/agent-test-tools</agent-test-tools.workingDirectory>
-        <agent-test-tools.repos>https://github.com/apache/skywalking-agent-test-tool.git</agent-test-tools.repos>
+        <agent-test-tools.repos>https://github.com/nisiyong/skywalking-agent-test-tool.git</agent-test-tools.repos>
```

Comment:
@wu-sheng After https://github.com/apache/skywalking-agent-test-tool/pull/49 merge, I will change the repo URL.